src/uct/ib/rc/accel/rc_mlx5.inl

```diff
@@ -226,6 +230,14 @@ uct_rc_mlx5_iface_rx_mp_context_from_hash(uct_rc_mlx5_iface_common_t *iface,
 
     last = cqe->byte_cnt & htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG);
 
+    if (ucs_likely(ucs_test_all_flags(cqe->byte_cnt,
+                                      htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG) |
+                                      htonl(UCT_IB_MLX5_MP_RQ_FIRST_MSG_FLAG)))) {
```

Comment:
can be simplified:

```suggestion
    if (ucs_likely(ucs_test_all_flags(cqe->byte_cnt,
                                      htonl(UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG |
                                            UCT_IB_MLX5_MP_RQ_FIRST_MSG_FLAG)))) {
```
or
```suggestion
    if (ucs_likely(ucs_test_all_flags(ntohl(cqe->byte_cnt),
                                      UCT_IB_MLX5_MP_RQ_LAST_MSG_FLAG |
                                      UCT_IB_MLX5_MP_RQ_FIRST_MSG_FLAG)))) {
```
i'd prefer the 2nd one

Reply:
th 2-d one is less efficient, because the first one makes calculations with the constant (compile-time optimization)

Reply:
ah, I see. make sense